src/odemis/acq/stitching/_tiledacq.py

```diff
@@ -1208,47 +1202,54 @@ def run(self):
 
         return da_rois
 
-def get_tiled_areas(
+def get_stream_based_bbox(
     pos: Dict[str, float],
     streams: List[Stream],
     tiles_nx: int,
     tiles_ny: int,
     overlap: float,
     tiling_rng: Dict[str, list],  # TODO: make optional, use axes range otherwise
-    selected_grids: List[str] = None,
-    sample_centers: Dict[str, Tuple[float]] = None,
-    whole_grid: bool = False,
-    rel_bbox: tuple = SAMPLE_USABLE_BBOX_TEM_GRID,
-) -> List[Tuple[float]]:
+) -> Optional[Tuple[float]]:
     """
-    Compute the bounding boxes of all areas to acquire.
+    Compute a bounding box based on streams, based on the number of x tiles, stream fov and overlap.
     :param pos: the current position of the stage
     :param streams: the streams to acquire
     :param tiles_nx: the number of tiles in the x direction
     :param tiles_ny: the number of tiles in the y direction
     :param overlap: the overlap between tiles (in percentage)
     :param tiling_rng: the tiling range along x and y axes as (xmin, ymin, xmax, ymax), or (xmin, ymax, xmax, ymin)
-    :param selected_grids: the grids to acquire
-    :param sample_centers: the centers of the sample grids
-    :param whole_grid: if True, the whole grid is acquired
+    :return: the bounding box (xmin, ymin, xmax, ymax in physical coordinates). If no area at all, returns None.
+    """
+    # Get al stream fov's, if any
+    fovs = [get_fov(s) for s in streams]
+    if not fovs:
+        # fall back to a small fov (default)
+        fov = DEFAULT_FOV
+    else:
+        # smallest fov
+        fov = tuple(map(min, zip(*fovs)))
+
+    return get_fov_based_bbox(
+        pos=pos,
+        fov=fov,
+        tiles_nx=tiles_nx,
+        tiles_ny=tiles_ny,
+        tiling_rng=tiling_rng,
+        overlap=overlap,
+    )
+
+
+def get_tiled_bboxes(
+    rel_bbox: tuple,
+    sample_centers: Dict[str, Tuple[float]] = None,
```

Comment:
remove default arg for sample_centers, not possible after refactor
```suggestion
    sample_centers: Dict[str, Tuple[float]],
```